src/stumpy.cr

```diff
@@ -0,0 +1,33 @@
+require "./pluto"
+require "stumpy_core"
+
+class Pluto::ImageRGBA
+  def initialize(canvas : StumpyCore::Canvas)
+    size = (@width = canvas.width) * (@height = canvas.height)
+    @red = Array(UInt8).new(size)
+    @green = Array(UInt8).new(size)
+    @blue = Array(UInt8).new(size)
+    @alpha = Array(UInt8).new(size)
+
+    canvas.each_row do |row|
+      row.each do |rgba|
+        @red << (rgba.r >> 8).to_u8
+        @green << (rgba.g >> 8).to_u8
+        @blue << (rgba.b >> 8).to_u8
+        @alpha << (rgba.a >> 8).to_u8
+      end
+    end
+  end
+
+  def to_stumpy : StumpyCore::Canvas
```

Comment:
I thought about naming it `to_canvas`, but that would assume that the only canvas would be Stumpy's. Canvas is a sufficiently generic word that I didn't want to be to, well, generic with this method name.